WalletWasabi.Gui/CommandLine/DropWalletCacheCommand.cs

```diff
@@ -0,0 +1,112 @@
+using NBitcoin;
+using Mono.Options;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using WalletWasabi.Helpers;
+using WalletWasabi.Services;
+using WalletWasabi.KeyManagement;
+
+namespace WalletWasabi.Gui.CommandLine
+{
+	internal class DropWalletCacheCommand : Command
+	{
+		public string WalletName { get; set; }
+		public Network Network { get; set; }
+		public Daemon Daemon { get; }
+		public bool ShowHelp { get; set; }
+
+		public DropWalletCacheCommand(Daemon daemon)
+			: base("dropwalletcache", "Drops the wallet cache for the specified wallet.")
+		{
+			Daemon = daemon;
+			Network = Network.Main;
+
+			Options = new OptionSet()
+			{
+				"usage: dropwalletcache --wallet:WalletName",
+				"",
+				"Drops the wallet cache for the specified wallet.",
+				"eg: ./wassabee dropwalletcache --wallet:MyWalletName --network:main",
+				"",
+				{ "w|wallet=", "The name of the wallet file.", x =>  WalletName = x },
+				{ "n|network=", "The network for the given file (main, test, reg).", x => Network = GetNetwork(x)},
```

Comment:
`GetNetwork(x)},` add space between `)` and `}`.

Reply:
```suggestion
				{ "n|network=", "The network for the given file (main, test, reg). Default=mainnet", x => Network = GetNetwork(x)},
```